.gitignore

```diff
@@ -19,3 +19,4 @@ yarn-error.log*
 *.njsproj
 *.sln
 *.sw*
+.scannerwork
```

Comment:
isn't `.sonar/` also needed?

Reply:
Hmm, when I tested it locally, it only created the `.scannerwork` folder. Besides, this will usually be executed by the CI server that won't commit anything afterwards. Anyhow, it wouldn't hurt anybody to add this to the `.gitignore` file, if you think it is needed. :)

Reply:
It is fine, I check on [this](https://www.gitignore.io/api/sonarqube) website but I have also seen that only `.scannerwork` is actually been created.